CHANGELOG.md

```diff
@@ -1,5 +1,38 @@
 # Change Log
 
+## [0.22.0](https://github.com/eclipse/lemminx/milestone/35?closed=1) (September 28, 2022)
+
+### Enhancements
+
+ * Suppress validation kinds based on file pattern. See [#1275](https://github.com/eclipse/lemminx/issues/1275).
+ * Fix CI badge, add some more badges. See [#1262](https://github.com/eclipse/lemminx/pull/1262).
+ * CodeAction for cvc-complex-type.2.4.b: "insert all expected" vs. "insert all possible". See [#1255](https://github.com/eclipse/lemminx/issues/1255).
+ * Use proxy config as HTTPS proxy as well. See [#1253](https://github.com/eclipse/lemminx/pull/1253).
```

Comment:
"Use the same proxy configuration for HTTPS"

Reply:
fixed